library/src/main/java/com/datatorrent/lib/io/fs/FileSplitterInput.java

```diff
@@ -96,7 +96,7 @@ public FileSplitterInput()
   {
     super();
     currentWindowRecoveryState = Lists.newLinkedList();
-    idempotentStorageManager = new IdempotentStorageManager.FSIdempotentStorageManager();
+    idempotentStorageManager = new IdempotentStorageManager.NoopIdempotentStorageManager();
```

Comment:
The reason to make this change is that if multiple ```FSIdempotentStorageManager```s  are used in an application then their state (on disk) collide.  The jira APEXMALHAR-2068 is opened to address that. 

The problem in having FSIdempotentStorageManager as default is that the users get bewildered because of in-correct behavior manifested by the bug mentioned above.

If the user sets FSIdempotentStorageManage explicitly then they can ensure to make the relative path unique as well.

Not setting any default entails additional step of setting an Idempotent storage manager even when the user doesn't care about idempotency.